dipy/tracking/mesh.py

```diff
@@ -0,0 +1,181 @@
+                                    triangles_weight=None, rand_gen=None):
+    """Generate random triangles_indices and trilinear_coord
+
+    Triangles_indices probability are weighted by triangles_weight,
+        for each triangles inside the given triangles_mask
+
+    Parameters
+    ----------
+    nb_triangles : int (n)
+        The amount of triangles in the mesh
+    nb_seed : int
+        The number of random indices and coordinates generated.
+    triangles_mask : [n] numpy array
+        Specifies which triangles should be chosen (or not)
+    triangles_weight : [n] numpy array
+        Specifies the weight/probability of choosing each triangle
+    random_seed : int
+        The seed for the random seed generator (numpy.random.seed).
+
+    Returns
+    -------
+    triangles_idx: [s] array
+        Randomly chosen triangles_indices
+    trilin_coord: [s,3] array
+        Randomly chosen trilinear_coordinates
+
+    See Also
+    --------
+    seeds_from_surface_coordinates, random_seeds_from_mask
+
+    Examples
+    --------
+    >>> from fury import io, utils   # skip if not has_fury
+    >>> mesh_polydata = io.load_polydata('mesh.vtk') # skip if not has_fury
+    >>> vts = utils.get_polydata_vertices(mesh_polydata) # skip if not has_fury
+    >>> triangles = utils.get_polydata_triangles(mesh_polydata) # skip if not has_fury
+    >>> mask = np.ones([len(triangles)])
+    >>> mask[0] = 0
+    >>> normal = np.cross(vts[triangles[:, 1]] - vts[triangles[:, 0]],
+    ... vts[triangles[:, 2]] - vts[triangles[:, 0]])
+    >>> triangles_area = 0.5*np.linalg.norm(normal,axis=-1)
+    >>> nb_seeds = 100
+    >>> idx, coord = random_coordinates_from_surface(len(triangles), nb_seeds,
+    ... mask, triangles_weight=triangles_area)
+    >>> seeds_pts = seeds_from_surface_coordinates(triangles, vts, idx,
+    ... coord)
+    """
+    # Compute triangles_weight in vts_mask
+    if triangles_mask is not None:
+        if triangles_weight is None:
+            triangles_weight = triangles_mask.astype(np.float)
+        else:
+            triangles_weight *= triangles_mask.astype(np.float)
+
+    # Normalize weights to have probability (sum to 1)
+    if triangles_weight is not None:
+        triangles_weight /= triangles_weight.sum()
+
+    # Set the random seed generator
+    np.random.seed(rand_gen)
+
+    # Choose random triangles
+    triangles_idx = \
+        np.random.choice(nb_triangles, size=nb_seed, p=triangles_weight)
+
+    # Choose random trilinear coordinates
+    # http://mathworld.wolfram.com/TrianglePointPicking.html
+    trilin_coord = np.random.rand(nb_seed, 3)
+    is_upper = (trilin_coord[:, 1:].sum(axis=-1) > 1.0)
+    trilin_coord[is_upper] = 1.0 - trilin_coord[is_upper]
+    trilin_coord[:, 0] = 1.0 - (trilin_coord[:, 1] + trilin_coord[:, 2])
+
+    return triangles_idx, trilin_coord
+
+
+def seeds_from_surface_coordinates(triangles, vts_values,
```

Comment:
After the import above, add the following decorator above the function: `@doctest_skip_parser`